packages/manager/apps/hpc-vmware-managed-vcd/src/pages/dashboard/datacentre/DatacentreDashboard.page.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { useNavigate, useParams, useResolvedPath } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { BreadcrumbItem } from '@/hooks/breadcrumb/useBreadcrumb';
+import VcdDashboardLayout from '@/components/dashboard/layout/VcdDashboardLayout.component';
+import { useManagedVcdDatacentre } from '@/data/hooks/useManagedVcdDatacentres';
+import useManagedVcdOrganization from '@/data/hooks/useManagedVcdOrganization';
+
+function DatacentreDashboardPage() {
+  const { id, vdcId } = useParams();
```

Comment:
**question:** Is it `vdcId` or `vcdId`?
```suggestion
  const { id, vcdId } = useParams();
```

Reply:
@anooparveti : it's really vdcId for virtual datacenter id